Src/Plugins/Tests.Security/EndToEndTests.cs

```diff
@@ -133,6 +133,8 @@ protected override string ConstructTestOutputFromInputResource(string inputResou
 
         protected void RunAllTests()
         {
+            string testVar = ProductTestDataDirectory;
```

Comment:
>string testVar = ProductTestDataDirectory; [](start = 12, length = 42)

remove this

Reply:
Done.

---
In reply to: [605122022](https://github.com/microsoft/sarif-pattern-matcher/pull/347#discussion_r605122022) [](ancestors = 605122022)